modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js

```diff
@@ -528,9 +528,9 @@ function serializeDefinition(
 
 		const xmlNode = XMLUtil.createObj(xmlType);
 
-		const tagNodeNameId = publishing ? 'name' : 'id';
+		const tagNodeNameId = exporting ? 'name' : 'id';
 
-		buffer.push(xmlNode.open, XMLUtil.create(`${tagNodeNameId}`, id));
+		buffer.push(xmlNode.open, XMLUtil.create(`${tagNodeNameId}`, item.id));
```

Comment:
@eliasantosf No need to use `item.id` because we already defined a `const id` above that receives the value of `item.id`.

Reply:
Done!